src/components/Tooltips/CanvasHoverTooltip.tsx

```diff
@@ -9,6 +9,7 @@ import { AnnotationState } from "../../colorizer/utils/react_utils";
 import { selectVectorConfigFromState } from "../../state/slices";
 import { FlexColumn, FlexRow } from "../../styles/utils";
 
+import { LabelType } from "../../colorizer/AnnotationData";
```

Comment:
![image](https://github.com/user-attachments/assets/50da19c0-7a48-45fb-8f21-2c4e8c7c2429)

Updates the hover tooltip to show current values